Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -1940,7 +1940,7 @@ void WebPage::loadRequest(LoadParameters&& loadParameters)
     }
 
     if (frame.get() == m_mainFrame.ptr())
-        corePage()->userInputBridge().loadRequest(WTFMove(frameLoadRequest));
+        m_mainFrame->coreLocalFrame()->userInputBridge().loadRequest(WTFMove(frameLoadRequest));
```

Comment:
We may want to null check all these new coreLocalFrame() calls to keep things from crashing while we're developing site isolation.